docs/getting-started.md

```diff
@@ -10,24 +10,16 @@ weight: 1
 
 Before you get started:
 
-- Install Git
-- Install Go `1.12.3` or later
-- This project uses [Go Modules](https://github.com/golang/go/wiki/Modules). Set `GO111MODULE=on` in your environment.
 - Kubernetes Cluster `1.13` or later (e.g. [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/))
 - [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) with version `1.13` or later
-- [Install Kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md) with version `2.0.3` or later
-
-## Go Modules
-
-> ⚠️ This project uses Go Modules. Due to the current state of code generation in [controller-tools](https://github.com/kubernetes-sigs/controller-tools) and [code-generator](https://github.com/kubernetes/code-generator), KUDO currently **must** be cloned into its `$GOPATH`-based location.
 
 ## Install KUDO into your cluster
 
-- Get KUDO repo: `git clone git@github.com:kudobuilder/kudo.git`
-- `cd kudo`
-- `kubectl create -f docs/deployment/00-prereqs.yaml`
-- `kubectl create -f docs/deployment/10-crds.yaml`
-- `kubectl create -f docs/deployment/20-deployment.yaml`
+- `kubectl create -f https://raw.githubusercontent.com/kudobuilder/kudo/master/docs/deployment/00-prereqs.yaml`
```

Comment:
This should be `make deploy | kubectl apply -f -` now.

Reply:
The RBAC related stuff in `docs/deployment/00-prereqs.yaml` is very different from what's generated by `make deploy`, so `make deploy | kubectl apply -f` is not equivalent to `kubectl create -f https://raw.githubusercontent.com/kudobuilder/kudo/master/docs/deployment/00-prereqs.yaml`